components/post/Resources.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { X } from "lucide-react";
+
+import { Button } from "../shared";
+
+const Resources = ({
+  useFieldArray,
+  register,
+  control,
+  errors,
+}: {
+  useFieldArray: any;
+  register: any;
+  control: any;
+  errors: any;
+}) => {
+  const { fields, append, remove } = useFieldArray({
+    name: "resources",
+    control,
+  });
+
+  return (
+    <section>
+      <div className="text-white-300 flex flex-col">
+        <h3 className="paragraph-3-medium text-white-500">RESOURCES & LINKS</h3>
+      </div>
+
+      <section className="mt-6">
+        {fields.map((field: { id: number }, index: number) => {
+          return (
+            <React.Fragment key={field.id}>
+              <div className="mb-2 flex gap-x-2">
+                <div className="flex w-full flex-col">
+                  <input
+                    type="text"
+                    className="placeholder:text-white-500 paragraph-3-regular bg-black-700 w-full rounded-md border-none p-3 focus:outline-none"
+                    placeholder="Label"
+                    {...register(`resources.${index}.label`)}
+                  />
+
+                  {errors.resources &&
+                    errors.resources[index]?.label?.message && (
+                      <span className="text-error-500 paragraph-3-regular">
+                        {errors.resources[index].label.message}
+                      </span>
+                    )}
+                </div>
+                <div className="flex w-full flex-col">
+                  <input
+                    type="text"
+                    className="placeholder:text-white-500 paragraph-3-regular bg-black-700 w-full rounded-md border-none p-3 focus:outline-none"
+                    placeholder="Resource Link"
+                    {...register(`resources.${index}.link`)}
+                  />
+                  {errors.resources &&
+                    errors.resources[index]?.link?.message && (
+                      <span className="text-error-500 paragraph-3-regular">
+                        {errors.resources[index].link.message}
+                      </span>
+                    )}
+                </div>
+                <button className="bg-black-700 h-11 rounded px-3">
+                  <X
+                    className="text-white-500"
+                    size={16}
+                    onClick={() => remove(index)}
+                  />
+                </button>
+              </div>
+            </React.Fragment>
+          );
+        })}
+      </section>
+
+      <div className="mt-4">
+        <Button
+          color="gray"
+          icon="plus"
```

Comment:
```suggestion
        <Button
          type="button"
          color="gray"
          icon="plus"
```

https://github.com/DDVVPP/GitNote/assets/4108484/4fd0bc52-2348-42d7-af5a-415159e4f60d